src/components/Modals/Nfts/NftModal.tsx

```diff
@@ -214,18 +213,18 @@ export const NftModal: React.FC<NftModalProps> = ({ zapperNft }) => {
           )}
         </Flex>
         <StatGroup>
-          {floorPriceEth && maybeFeeAsset && (
+          {floorPrice && maybeFeeAsset && (
             <Stat>
               <StatLabel>{translate('nft.floorPrice')}</StatLabel>
               <StatNumber>
-                {floorPriceEth} {maybeFeeAsset.symbol}
+                {floorPrice} {maybeFeeAsset.symbol}
               </StatNumber>
             </Stat>
           )}
-          {lastSaleEth && (
+          {price && (
             <Stat>
               <StatLabel>{translate('nft.lastPrice')}</StatLabel>
-              <StatNumber>{lastSaleEth} ETH</StatNumber>
+              <StatNumber>{price} ETH</StatNumber>
```

Comment:
While at it, this is wrong
```suggestion
              <StatNumber>{price} {maybeFeeAsset.symbol}</StatNumber>
```